### ISSUE TITLE
Unstable / wobbly video using any non-pass through resolutions

### ISSUE THREAD

**Issue by mambaJB** [ramapcsx2/gbs-control#279]
Firstly, this is an awesome project! - thank you rama and everyone involved, much respect !

I have setup my GBS Control (GBS8200 v4.0), however when using any of the upscale resolutions, the output video is extremely noisy and or wobbly, with pass through, everything is perfect, I have noticed that the frequency does change when using the different resolutions, not sure if this is normal so for example, pass-through is (640x480 31.5kHZ 60Hz) and moving to 720p it’s (1260x720 61.5Khz 60Hz)

I am using Component input 480p (I have read that 480p seems to struggle, is this the struggle the one issue was referring to?) and have tried a PC on VGA - similar result, the following mods have been done:
* Pots removed and bridged
* 4 Stacked ceramic caps @10uF each
* Removed C11
* External clock gen (3.3v C48 power)
* Tried 100ohm resistor on GND and S for VGA
* Tried direct VGA output, and lagless VGA2HDMI

I might have missed a mod to do or something I have yet to try, any pointers into trying to resolve this ? Let me know if there’s any additional information or steps I need to provide to help better debug.
![C2A4D1EA-5DB1-4E7A-8E1B-0011369037F2](https://user-images.githubusercontent.com/92269097/136710756-4c9366b2-e8d8-4c46-a981-69352576d0cb.jpeg)

**Comment by mambaJB** [comment]
Yes correct, different input signals, I started with a Xbox component input, pass through works fine at 480p and with the 720p default pass through, as soon as Input is 480p and I set an upscale of 1280x960 or 720p I get those affects, Xbox is a 1.6b revision, i then thought let me try VGA, same result, passthrough works wonders but any upscaling done causes these weird effects.

I do have a PS1 and 2 to try but the component cables are still in the mail, do fear perhaps I may run into the same issue.
Anything to try.?

**Comment by mambaJB** [comment]
Resurrecting this as I have managed to fix the issue, so that anyone else browsing that has the problem I experienced can at least have some troubleshooting start point.

The fix was 2 fold: 1 being that there were cold solder joins between the Ground pins and the NodeMCU, as well as cold solder joints between the Myson MTV chip and the ESP NodeMCU - my solder wire (tin wire) may have been old or was not labeled correctly and did not contain flux (even though I used added flux with the 60/40 wire), reflowing the connector wires with nice new tin improved the signal and removed a lot of the wobble.

The 2nd issue was indeed the clock generator, I had to remove the clock generator and image is now stable completely, oddly, the clock generator pin, I left it high without soldering it back to the pad and so far so good! I have stable video and signal, upscalling is beautiful and works with the generic cables of the xbox and the retro PC! - I do intend to buy a new clock generator in the future.

Hope this helps someone down the years to come ! Thanks again Rama for the great work !
also to note - I have not updated the firmware, its the same since I opened the issue.

**Comment by mambaJB** [comment]
Resurrecting this as I have managed to fix the issue, so that anyone else browsing that has the problem I experienced can at least have some troubleshooting start point.

The fix was 2 fold: 1 being that there were cold solder joins between the Ground pins and the NodeMCU, as well as cold solder joints between the Myson MTV chip and the ESP NodeMCU - my solder wire (tin wire) may have been old or was not labeled correctly and did not contain flux (even though I used added flux with the 60/40 wire), reflowing the connector wires with nice new tin improved the signal and removed a lot of the wobble.

The 2nd issue was indeed the clock generator, I had to remove the clock generator and image is now stable completely, oddly, the clock generator pin, I left it high without soldering it back to the pad and so far so good! I have stable video and signal, upscalling is beautiful and works with the generic cables of the xbox and the retro PC! - I do intend to buy a new clock generator in the future.

Hope this helps someone down the years to come ! Thanks again Rama for the great work !
also to note - I have not updated the firmware, its the same since I opened the issue.
